app/views/spree/admin/payments/source_forms/_braintree.html.erb

```diff
@@ -0,0 +1,64 @@
+<fieldset data-id='credit-card' class="no-border-bottom">
```

Comment:
Nothing, the only reason it is here is because of the change in `method_type` for the gateway. This broke the drop in form that was being used in the backend.